src/init/lockdown.rs

```diff
@@ -63,11 +63,11 @@ pub fn calculate_lockdown_per_day(
         }
     }
 
-    // Find the mean probability of staying at home, over the entire population
-    let mean_pr_home = population
+    // Find the mean decrease of time spent outside of home, over the entire population
+    let mean_pr_home_tot = population
         .people
         .iter()
-        .map(|person| person.time_use.home as f32)
+        .map(|person| person.time_use.home_tot as f32)
```

Comment:
Close! I named it home_total: https://github.com/dabreegster/uatk-spc/blob/f4c6a913ad89d604e57664943e4419580a12e5e5/synthpop.proto#L150